GitHubPolicy.md

```diff
@@ -0,0 +1,177 @@
+
+* Check translations. 
+    > Make sure all strings are translation strings (not hardcoded). As well make sure that string will make sense for the translator who will not have any context. If it is ambigous, ask the author of the PR to add a description. You can learn more about translations [here]()
+
+* Check for any unecessary comments.
+    > If the comment does not express something meaningful to another developer, ask the author to change or delete the comment. This includes any code that is commented out. If you do not know why code is commented out, ask the author to leave a comment explaining why it is commented out, or to erase it entirely. 
+
+* Does the code make sense? 
+    > If there is anything that doesn't make sense, leave a comment asking for clarification. After clarification, can you suggest another way to write the code, that makes it more human readable? If not, ask the contributor to add a comment above giving a brief explanation
+
+* Squash and merge.
+    > When the PR has been approved, (squash and merge)[] instead of just merging. This makes the history show by order or PR instead of commit
+
+
+## Giving Feeback
+* Give constructive feedback
+    > The feedback should either be asking a question, or giving suggestions about how to fix something. For example, if something is unclear, instead of saying "This is unclear", ask the author "Can you explain what you are doing here?", or say "This is unclear, here is how you can make it more obvious..."
+
+* Be descriptive while being concise
+    > Describe the issue, while making it very clear what you are describing. Steer away from any unnecessary language.
+
+
+* Be Empathetic
+    > Give feedback that humanizes the author, and recognizes the work that has gone into it. Try to use gentle language. Ask questions, when you are unsure, instead of assuming that something is done incorecctly. Use "I" instead of "you" when giving feedback. For example, dont say "You did this wrong", say "I would have done this x way, can you explain why it was done this way?". 
+
+
+## Template
+> # Header
+> Give the header a meaningful name that describes what you are doing. For Example, "New Page" is too broad of a name that doesn't specifiy what bug fix has been done. Instead use "Intro Page for App tutorial Created." Try to be descriptive, while also being concise.
+>
+># Description
+> Describe What exactly has been done:
+> Give a highlevel overview of what work has been done.
+>
+> If there has been a change in the UI, also describe how to view this change in the app. When necessary give step by step instructions on what steps to recreate in order to view this change.
+>
+> If this is a bug fix, describe what the original bug was.
+>
+> Add Screen shots of the new UI
+>
+> # Wireframes
+> Include link to wireframes here
+>
+> # To Do
+> Include any tasks that still need to be completed that cannot be addressed in this PR
+GIVE EXAMPLES HERE
+
+## How To
```

Comment:
this might be an iffy section because of the git interface they're using (e.g. GitHub's desktop app, git cli, gh cli client, etc). generally would point to official documentation about each thing if available (i'm sure GitHub has some pages dedicated to some of these)

Reply:
Right, so I think the majority of people will be using browser, and if not, then I think they can copy and paste it off this document. I think there is a lot of value to having a template.

Reply:
so would you just list CLI commands in this section? My point is that you can have many variations of each step depending on the interface that's being used, which will make this document pretty verbose if you try to document all of them.